hikari/api/special_endpoints.py

```diff
@@ -1006,7 +1017,7 @@ def set_id(self: _T, id_: undefined.UndefinedOr[snowflakes.Snowflakeish], /) ->
 
     @abc.abstractmethod
     def set_default_permission(self: _T, state: undefined.UndefinedOr[bool], /) -> _T:
-        """Whether this command should be enabled by default (without any permissions).
+        """Set whether this command should be enabled by default (without any permissions).
```

Comment:
I am not sure, does this disable commands regardless of any permission (including disabling for admins here)?

Reply:
Yep, but this is deprecated and will be removed. I havent modified the logic for it in this PR

Reply:
Ah I see, thanks for letting me know!